elect

```diff
@@ -0,0 +1,21 @@
+=> [[32m#<Meal:0x00000000058ce808[0m
```

Comment:
what a heck is that? )

move `public/uploads` => `.gitinore` to avoid adding your uploaded test files to repository

Reply:
remove file